flatgfa/Cargo.toml

```diff
@@ -17,4 +17,5 @@ memmap = "0.7.0"
 num_enum = "0.7.3"
 rayon = "1.10.0"
 tinyvec = "1.8.0"
+rand = "0.8"
```

Comment:
Because `rand` is only used in the tests, and not in the final code itself, can you please make this a dev dependency? https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html